syftr/cli.py

```diff
@@ -0,0 +1,192 @@
+import logging
+import os
+from pathlib import Path
+
+import click
+
+import syftr.scripts.system_check as system_check
+from syftr.api import Study, SyftrUserAPIError, stop_ray_job
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+
+def _get_study(config_path: str, study_name: str):
+    if (config_path and study_name) or (not config_path and not study_name):
+        raise click.UsageError(
+            "Provide exactly one of:\n"
+            "  • a positional CONFIG_PATH to launch a new study,\n"
+            "  • OR `--name STUDY_NAME` to attach to an existing study.\n"
+            "(You passed: config_path=%r, study_name=%r)" % (config_path, study_name)
+        )
+    if config_path:
+        return Study.from_file(Path(config_path))
+    else:
+        return Study.from_db(study_name)
+
+
+@click.group()
+def main():
+    """syftr command‐line interface for running and managing studies."""
+    pass
+
+
+@main.command()
+def check():
+    """
+    syftr check
+    ---
+    Checks the system for required dependencies and configurations.
+    """
+    system_check.check()
+
+
+@main.command()
+@click.argument(
+    "config_path",
+    type=click.Path(exists=True, dir_okay=False),
+    required=False,
+)
+@click.option(
+    "--name",
+    "study_name",
+    type=str,
+    help=(
+        "Name of an existing study to attach to or resume. "
+        "If you supply a positional CONFIG_PATH, --name must be omitted."
+    ),
+)
+@click.option(
+    "--follow/--no-follow",
+    default=False,
+    help="Stream logs until the study completes.",
+)
+def run(config_path: str, study_name: str, follow: bool):
+    """
+    syftr run [CONFIG_PATH | --name STUDY_NAME] [--follow]
+
+    • To launch a new study from YAML:
+        syftr run path/to/config.yaml [--follow]
```

Comment:
```suggestion
        syftr run path/to/study-config.yaml [--follow]
```

Don't confuse people between system config and study config